libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -674,8 +677,29 @@ bool PbftManager::stateOperations_() {
   LOG(log_tr_) << "PBFT current step is " << step_;
 
   // Periodically verify unverified votes
-  vote_mgr_->verifyVotes(round, sortition_threshold_, getDposTotalVotesCount(),
-                         [this](auto const &addr) { return dposEligibleVoteCount_(addr); });
+  vote_mgr_->verifyVotes(round, [this](auto const &v) {
```

Comment:
maybe this is kind of tediousness from my side, but I think this is little easier to understand 
```cpp
  vote_mgr_->verifyVotes(round, [this](auto const &v) {
    auto dpos_votes_count = dposEligibleVoteCount_(v->getVoterAddr());
    if (!dpos_votes_count) {
      return false;
    }

    auto total_votes_count = getDposTotalVotesCount();
    auto threshold = sortition_threshold_;

    if (v->getStep() == 1) {
      // We need to handle propose_vote_type
      // verification differs for step 1
      dpos_votes_count = 1;
      total_votes_count = getDposTotalAddressCount();
      threshold = std::min(getDposTotalAddressCount(), COMMITTEE_SIZE);
    }

    try {
      v->validate(dpos_votes_count, total_votes_count, sortition_threshold_);
    } catch (const std::logic_error &e) {
      LOG(log_wr_) << e.what();
      return false;
    }
    return true;
  }
```

Reply:
definitely easier to read :)